addons/web_editor/static/src/components/media_dialog/media_dialog.js

```diff
@@ -241,6 +247,11 @@ export class MediaDialog extends Component {
                     return true;
                 }));
             }
+            for (const candidateFunc of TABS[this.state.activeTab].Component.mediaExtraClasses) {
+                if (typeof(candidateFunc) === 'function') {
+                    element.classList.add(candidateFunc(this.props));
+                }
+            };
```

Comment:
Useless `;` here, you have to install a linter :wink: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines (not sure if those tools work on website files though. If not, until they do, any basic linting will be fine).